docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml

```diff
@@ -589,33 +597,299 @@ depending on where they wind up.-->
+                button.</para>
+              </listitem>
+
+              <listitem>
+                <para>Press <emphasis role="bold">OK</emphasis> when prompted
+                to confirm.</para>
+              </listitem>
+            </orderedlist>
+          </sect4>
+
+          <sect4 id="GetPart">
+            <title>Get Part</title>
+
+            <para><figure>
+                <title>Get Part</title>
+
+                <mediaobject>
+                  <imageobject>
+                    <imagedata fileref="../../images/ECLWaGetPart01.jpg"
+                               vendor="ECLWatchSS" />
+                  </imageobject>
+                </mediaobject>
+              </figure></para>
+
+            <para>Press the <emphasis role="bold">Get Part</emphasis> button
+            to view the contents of the selected part. <figure>
+                <title>Package Part Contents</title>
+
+                <mediaobject>
+                  <imageobject>
+                    <imagedata fileref="../../images/ECLWaGetPart02.jpg"
+                               vendor="ECLWatchSS" />
+                  </imageobject>
+                </mediaobject>
+              </figure></para>
+          </sect4>
+        </sect3>
+
+        <sect3 id="Packagemap_Validation">
+          <title>Validate Package Map</title>
+
+          <para>The <emphasis role="bold">Validate Package Map</emphasis> tab
+          is used to validate active package maps. The <emphasis
+          role="bold">Validate Package Content</emphasis> tab is used to
+          validate package map content that is not yet loaded. To validate an
+          active package map:</para>
+
+          <para><figure>
+              <title>Validate Package Maps</title>
+
+              <mediaobject>
+                <imageobject>
+                  <imagedata fileref="../../images/ECLWaPkMgr.jpg" />
+                </imageobject>
+              </mediaobject>
+            </figure></para>
+
+          <orderedlist>
+            <listitem>
+              <para>Select the <emphasis role="bold">Validate Package
+              Map</emphasis> tab</para>
+            </listitem>
+
+            <listitem>
+              <para>Choose the <emphasis role="bold">Target</emphasis> and
+              <emphasis role="bold">Process</emphasis> from the drop lists on
+              the <emphasis role="bold">Validate Package Map</emphasis>
+              tab.</para>
+            </listitem>
+
+            <listitem>
+              <para>Press the <emphasis role="bold">Validate</emphasis> button
+              to validate the package map.</para>
+            </listitem>
+          </orderedlist>
+
+          <para>The result is shown on the <emphasis role="bold">Validate
+          Active Package Map</emphasis> tab. You can validate any package map,
+          active or inactive.</para>
+
+          <para>To validate any package map:</para>
```

Comment:
Not sure on exact wording, but the key is these are packagemaps that aren't yet on the system.

Something like "To validate an external packagemap"
and/or "a packagemap not yet uploaded onto the environment."